hw_lesson8.py

```diff
@@ -0,0 +1,82 @@
+"""
+Task 1
+A simple function.
+
+Create a simple function called favorite_movie, which takes a string containing the name of your favorite movie.
+The function should then print “My favorite movie is named {name}”.
+"""
+
+film_name = input('Now, my favorite movie is: ')
+
+
+def favorite_movie(film_name: str):
+    print(f'My favorite movie is named "{film_name}"')
+
+
+favorite_movie(film_name)
+
+"""
+Task 2
+Creating a dictionary.
+
+Create a function called make_country, which takes in a country’s name and capital as parameters.
+Then create a dictionary from those two, with ‘name’ as a key and ‘capital’ as a parameter.
+Make the function print out the values of the dictionary to make sure that it works as intended.
+"""
+
+country = str(input('Enter country name: '))
```

Comment:
You don't need to cast to a string, because input already returns `str()` type. :)